### ISSUE TITLE
VError: pack collection: entrancecourses failed: noTimeout cursors are disallowed in this atlas tier

### ISSUE THREAD

**Issue by ganesh7396-hub** [lihaibh/mongodb-snapshot#29]
while backup atlas tier local facing .iam using  mongo db clould 
could not write collection data into: type: Local FileSystem Connector, path: ./backup.tar due to error:
VError: pack collection: entrancecourses failed: noTimeout cursors are disallowed in this atlas tier
    at CatchSubscriber.selector (D:\MY WORKING DIRECTORY\DEVELOPMENT\node_modules\mongodb-snapshot\dist\connectors\filesystem\FileSystemDuplexConnector.js:21:73)

   {
    ok: 0,
    code: 8000,
    codeName: 'AtlasError'
  },
  jse_info: {}
}
removing: "type: Local FileSystem Connector, path: ./backup.tar" because an error was thrown during the transfer
![GIT issues](https://user-images.githubusercontent.com/59719566/129398183-2fc0c956-8206-41d6-b1e1-ea5b25e9edc0.png)

**Comment by lihaibh** [comment]
solved and merged at: #28
use version: `1.4.0`

**Comment by levimykel** [comment]
Hello. I'm getting this same exact error, even though I'm using version `1.4.0`.

Is there a trick to getting this to work?

```
could not write collection data into: type: Local FileSystem Connector, path: ./backup.tar due to error:
VError: pack collection: transactions failed: noTimeout cursors are disallowed in this atlas tier
    at CatchSubscriber.selector (/workspace/node_modules/mongodb-snapshot/dist/connectors/filesystem/FileSystemDuplexConnector.js:21:83)
    at CatchSubscriber.error (/workspace/node_modules/rxjs/internal/operators/catchError.js:48:31)
    at /workspace/node_modules/mongodb-snapshot/dist/connectors/filesystem/FileSystemDuplexConnector.js:152:26
    at Sink.<anonymous> (/workspace/node_modules/tar-stream/pack.js:175:14)
    at Sink.f (/workspace/node_modules/once/once.js:25:25)
    at Sink.onfinish (/workspace/node_modules/end-of-stream/index.js:31:27)
    at Sink.emit (events.js:314:20)
    at finishMaybe (/workspace/node_modules/tar-stream/node_modules/readable-stream/lib/_stream_writable.js:624:14)
    at endWritable (/workspace/node_modules/tar-stream/node_modules/readable-stream/lib/_stream_writable.js:643:3)
    at Sink.Writable.end (/workspace/node_modules/tar-stream/node_modules/readable-stream/lib/_stream_writable.js:571:22) {
  jse_shortmsg: 'pack collection: transactions failed',
  jse_cause: MongoError: noTimeout cursors are disallowed in this atlas tier
      at MessageStream.messageHandler (/workspace/node_modules/mongodb-snapshot/node_modules/mongodb/lib/cmap/connection.js:299:20)
      at MessageStream.emit (events.js:314:20)
      at processIncomingData (/workspace/node_modules/mongodb-snapshot/node_modules/mongodb/lib/cmap/message_stream.js:144:12)
      at MessageStream._write (/workspace/node_modules/mongodb-snapshot/node_modules/mongodb/lib/cmap/message_stream.js:42:5)
      at doWrite (_stream_writable.js:403:12)
      at writeOrBuffer (_stream_writable.js:387:5)
      at MessageStream.Writable.write (_stream_writable.js:318:11)
      at TLSSocket.ondata (_stream_readable.js:718:22)
      at TLSSocket.emit (events.js:314:20)
      at addChunk (_stream_readable.js:297:12) {
    ok: 0,
    code: 8000,
    codeName: 'AtlasError'
  },
  jse_info: {}
}
removing: "type: Local FileSystem Connector, path: ./backup.tar" because an error was thrown during the transfer
```
